app/assets/stylesheets/custom.css.scss

```diff
@@ -1,7 +1,28 @@
 .font-size-22 {
   font-size: 22px;
 }
-
+.notice {
+  display: block;
+  padding: 9.5px;
+  margin: 0 0 10px;
+  font-size: 23px;
+  line-height: 1.42857143;
+  color: #383;
+  background-color: #f5f5f5;
```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.